src/main/java/org/jabref/logic/search/retrieval/LuceneSearcher.java

```diff
@@ -134,7 +134,8 @@ private SearchResults getSearchResults(TopDocs topDocs, StoredFields storedField
         return searchResults;
     }
 
-    private static String getFieldContents(Document document, SearchFieldConstants field) {
```

Comment:
Why this linebreak? Revert?

Reply:
Intellji issue:
![idea64_MyrGYmpjmI](https://github.com/user-attachments/assets/39db8821-4d2c-4128-8569-d6eb71a33684)